go/enclave/db/rawdb/accessors_chain.go

```diff
@@ -16,17 +16,12 @@ import (
 )
 
 func ReadRollup(db ethdb.KeyValueReader, hash gethcommon.Hash) (*core.Rollup, error) {
-	height, err := ReadHeaderNumber(db, hash)
```

Comment:
There were a bunch of methods where we were keying stuff in the DB by hash + number. This wasn't needed - just using the hash was sufficient.